validator_client/src/config.rs

```diff
@@ -261,6 +267,19 @@ impl Config {
                 .collect::<Result<_, _>>()?;
         }
 
+        /*
+         * Beacon node fallback
+         */
+
+        config.beacon_node_fallback.disable_run_on_all = cli_args.get_flag("disable-run-on-all");
```

Comment:
deprecated flag

Reply:
Fixed in [453e003](https://github.com/sigp/lighthouse/pull/4393/commits/453e003f926e27c09b5a04be7c18f7f6efd8f99f)